src/groups/mqb/mqbblp/mqbblp_queueconsumptionmonitor.t.cpp

```diff
@@ -503,19 +498,7 @@ TEST_F(Test, putAliveIdleWithConsumer)
     ASSERT_EQ(logObserver.records().size(), ++expectedLogRecords);
     ASSERT(mwctst::ScopedLogObserverUtil::recordMessageMatch(
         logObserver.records().back(),
-        "ALARM \\[QUEUE_CONSUMER_MONITOR\\].*It currently has 2 consumers",
```

Comment:
Refactored the test by simplifying logic and removing consumers, but still need `Test::d_queueState` and its dependencies, because it is required for `d_monitor`.